inspirehep/modules/authors/tasks.py

```diff
@@ -27,16 +27,14 @@
 import os
 
 from flask import current_app, url_for
-from sqlalchemy.orm.exc import NoResultFound
-
+from inspire_dojson.utils import strip_empty_values
+from inspire_schemas.api import validate
 from invenio_accounts.models import User
 from invenio_oauthclient.models import UserIdentity
+from sqlalchemy.orm.exc import NoResultFound
```

Comment:
I don't like the import changes you made in all this PR : /

Did you use an automatic tool for them?

Reply:
I don't know to be honest why this happened...hmmm ok let me check

Reply:
sorry about that

Reply:
No need to be sorry about that, because this gives me a chance to explain something!

The reason you should resist doing these large scale refactorings while you are modifying a tiny bit of code is that they might generate a conflict in a lot of other PRs, and conflict resolution always require a little bit of care. So in exchange for a little improvement in the code base you generate potential errors for everyone else, which is not a good trade off.